src/apps/ipv4/arp.lua

```diff
@@ -144,11 +145,17 @@ function ARP:new(conf)
    return setmetatable(o, {__index=ARP})
 end
 
+function ARP:arp_resolving (ip)
+   print(("ARP: Resolving '%s'"):format(ipv4:ntop(self.next_ip)))
+   local key = {resource='external-interface', alarm_type_id='arp-resolution'}
+   alarm_codec.raise_alarm(key)
```

Comment:
OK, I didn't add the boolean cause the code has no effect if the alarm channel doesn't exist. But if we make alarm_codec.get_channel() to create the alarms_channel I think a flag will be required.